sources/engine/Stride.UI/UIComponentProcessor.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) .NET Foundation and Contributors (https://dotnetfoundation.org/ & https://stride3d.net)
+// Distributed under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using Stride.Engine;
+using Stride.Games;
+
+namespace Stride.UI;
+
+/// <summary>
+/// The processor in charge of updating entities that have <see cref="UIComponent"/>s.
+/// </summary>
+public class UIComponentProcessor : EntityProcessor<UIComponent, UIDocument>
```

Comment:
The reason it is seperate is because you can just directly add a `UIDocument` to the `UISystem`, and a `RenderUIDocument` to the renderer, to get UI without using the entity system at all. Previous work was done to decouple rendering of the UI from the ECS system. So to avoid losing this ability, I seperated the entity processing out as well. Additionally, the `UISystem` is manually created and needs to run in the editor.

```cs
var uiSystem = Services.GetService<UISystem>();
var sceneSystem = Services.GetService<SceneSystem>(); // Needed to get the RenderSystem, afaik.
var renderSystem = sceneSystem.GraphicsCompositor.RenderSystem;

var uiDocument = new UIDocument()
{
  // Set Page and other properties here...
};
var renderUiDocument = new RenderUIDocument()
{
  // Copy same properties from UIDocument, along with settng the position matrix.
};

uiSystem.AddDocument(uiDocument);
renderSystem.AddRenderObject(renderUiDocument);
```